pkg/option/config.go

```diff
@@ -621,6 +622,14 @@ const (
 	// K8sNamespaceName is the name of the K8sNamespace option
 	K8sNamespaceName = "k8s-namespace"
 
+	// AgentNotReadyNodeTaintPrefixDefault hold the default value of
+	// AgentNotReadyNodeTaintPrefix
+	AgentNotReadyNodeTaintPrefixDefault = ""
```

Comment:
is this default param used anywhere ?

Reply:
I actually meant to use that since it didn't look like `viper.GetString` will return a default empty-string value. I'm a bit confused though as I'm trying to figure out the behaviors. For example, `k8s-namespace` doesn't appear to have a default value set anywhere except in `bugtool`. I'll start work on an edit to use that default if the config value isn't set, but if you happen to know if there's a standard way to set those defaults I'd welcome a pointer.